client/src/utils/getToolbarAppParams.test.ts

```diff
@@ -0,0 +1,49 @@
+import { getCompressParams } from './getCompressParams';
```

Comment:
good test 👍 

could this test be split into two files
* getCompressParams.test.ts
* getExtractParams.test.ts